src/Compute/v2/Models/Hypervisor.php

```diff
@@ -62,6 +62,15 @@ class Hypervisor extends OperatorResource implements
     public $vcpus_used;
 
     /** @var array */
+    public $cpu_info;
+
+    /** @var int */
+    public $current_workload;
+
+    /** @var int */
+    public $disk_available_least;
```

Comment:
I'd rather rename everything and release in a minor version bump (v2.1.0). We'd need to document warnings that fields were renamed. Ideally this would be encapsulated in a breaking change release (v3) but I think that's overkill for one class.

@clwells Would you mind updating these fields to camelCame?